src/Catty/DataModel/Bricks/Arduino/ArduinoSendDigitalValueBrick.m

```diff
@@ -67,10 +67,10 @@ - (NSString*)description
 - (BOOL)isEqualToBrick:(Brick*)brick
 {
     if (![self.pin isEqualToFormula:((ArduinoSendDigitalValueBrick*)brick).pin]) {
-        return NO;
+        NO;
```

Comment:
Please have a look at the Swift documentation (https://docs.swift.org/swift-book/LanguageGuide/Functions.html):
> If the entire body of the function is a single expression, the function implicitly returns that expression.

The first part is very important: _If the entire body of the function is a single expression_. In this case there is no implicit return, since the body contains more than one single expression. Removing the _return_ from your line of code will not result in an actual return at all and thus change the code execution path in an undesired way.

Please revise your pull request.

Reply:
One additional remark: this of course only applies to Swift code, not ObjC.